flexget/plugins/api_tvmaze.py

```diff
@@ -119,6 +203,11 @@ def update(self, series, session):
         self.last_update = datetime.now()
 
         self.genres[:] = get_db_genres(series.genres, session)
+        if series.cast and series.cast.people:
+            self.actors[:], self.characters[:] = get_db_actors_and_characters(series.cast.people, session)
```

Comment:
I can look into it if you prefer but it currently works. You can check out the last test added.